src/Seq.App.HttpRequest/HttpRequestMessageFactory.cs

```diff
@@ -19,13 +20,28 @@ class HttpRequestMessageFactory
         readonly List<(string, string)> _headers;
         readonly System.Text.Encoding _utf8 = new UTF8Encoding(false);
 
-        public HttpRequestMessageFactory(string urlTemplate, HttpMethod method, string? bodyTemplate, string? mediaType, List<(string, string)> headers)
+        public HttpRequestMessageFactory(string urlTemplate, HttpMethod method, string? bodyTemplate, bool bodyIsTemplate, string? mediaType, List<(string, string)> headers)
         {
             if (urlTemplate == null) throw new ArgumentNullException(nameof(urlTemplate));
+            
             _mediaType = mediaType;
             _headers = headers;
+            
             _url = new ExpressionTemplate(urlTemplate, encoder: new TemplateOutputUriEncoder());
-            _body = bodyTemplate != null || mediaType != null ? new ExpressionTemplate(bodyTemplate ?? "") : null;
+
+            if (bodyTemplate != null || mediaType != null)
+            {
+                var bodyTemplateContent = bodyTemplate ?? "";
+                if (!bodyIsTemplate)
+                    bodyTemplateContent = ExpressionTemplate.EscapeLiteralText(bodyTemplateContent);
+
+                _body = new ExpressionTemplate(bodyTemplateContent);
```

Comment:
This is a little bit round-about, but keeps the configuration side of things (options like "body is template") contained.

Reply:
if (!bodyIsTemplate) 
  bodyTemplate ... 

Can we rename 'bodyTemplate'? 'body' since that is what it is called in the form?

Reply:
Would the boolean logic read better in reverse?

```
if (bodyTemplate == null && mediaType == null) {
  _body = null;
} else {
  ...
}
```

Reply:
Much nicer! Done, thanks 👍

Reply:
If also inverted 👍

Reply:
(Alt-Enter, Invert If, commit, push :-D )